sos/policies/arch.py

```diff
@@ -0,0 +1,48 @@
+from sos.policies import PackageManager, LinuxPolicy
```

Comment:
This needs the copyright header that is present in all other sos files:

```
# This file is part of the sos project: https://github.com/sosreport/sos
#
# This copyrighted material is made available to anyone wishing to use,
# modify, copy, or redistribute it subject to the terms and conditions of
# version 2 of the GNU General Public License.
#
# See the LICENSE file in the source distribution for further information.
```


Optionally, add a copyright line above it if you want to. E.G. `# Copyright (C) 2019 Optional Company, Your Name <your email>
`